sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -278,15 +278,27 @@ def _build_sentinel(setting: Union[str, Tuple[str, str]]) -> Tuple[str, str]:
 def _is_retryable_error(error: HttpResponseError) -> bool:
     """Determine whether the service error should be silently retried after a backoff period, or raised.
     Don't know what errors this applies to yet, so just always raising for now.
```

Comment:
nit: Do we still need this comment?

Reply:
Nope I don't think so - this was from before I knew which status code was the throttling error (429).